tests/bootstrap-api-coverage.php

```diff
@@ -1,7 +1,11 @@
 <?php
+
 /**
- * @copyright 2015-2016 Contentful GmbH
+ * This file is part of the contentful.php package.
```

Comment:
I'd like to avoid what I did in the CMA (with a single commit changing everything) but instead changing the header whenever the file is being changed...
This also needed updating, as it was saying 2015-2016, so it was two birds with one stone.